draft-ietf-core-senml.md

```diff
@@ -1244,15 +1244,7 @@ simplifies implementation of very simple system and does not impose
 any significant limitations as all this data is meant for machine to 
 machine communications and is not meant to be human readable. 
 
-Security considerations: Sensor data can contain a wide range of 
-information ranging from information that is very public, such the 
-outside temperature in a given city, to very private information that 
-requires integrity and confidentiality protection, such as patient 
-health information. This format does not provide any security and 
-instead relies on the transport protocol that carries it to provide 
-security. Given applications need to look at the overall context of 
-how this media type will be used to decide if the security is 
-adequate. 
+Security considerations: See {{sec-sec}}.
```

Comment:
Since this is text that goes into the media type registry, this would need to be "See {{sec-sec}} of RFCthis".